app_utils/sections/experiment.py

```diff
@@ -16,6 +17,10 @@
 from h2o_wave import Q, data, ui
 from jinja2 import Environment, FileSystemLoader
 from sqlitedict import SqliteDict
+from auto_gptq import AutoGPTQForCausalLM, BaseQuantizeConfig
+
+# FIXME: remove?
+from datasets import load_dataset, Dataset
```

Comment:
May not need this import after getting `cfg.dataset` to work for extracting all of the examples